test/jest/mocks.js

```diff
@@ -148,3 +152,39 @@ export const getInitialSendStateWithExistingTxState = (draftTxState) => ({
     },
   },
 });
+
+export function createMockInternalAccount({
+  address = '0xd5e099c71b797516c10ed0f0d895f429c2781111',
```

Comment:
If the intent is to centralise these mocks to avoid duplication, is it worth putting this address in a constant such as `ACCOUNT_MOCK` so we can assert against it later?